src/explorer/WebAppTreeItem.ts

```diff
@@ -126,3 +127,20 @@ export class WebAppTreeItem extends SiteTreeItem {
         return await fs.readFile(templatePath, 'utf8');
     }
 }
+
+export class InvalidWebAppTreeItem implements IAzureTreeItem {
+    public static contextValue: string = 'file';
```

Comment:
I assume you copied this from the FileTreeItem haha? Please make the context value something like 'invalidWebApp' or 'invalidAppService'

Reply:
Lol, yeah, my bad... copy/paste got me. Renamed to 'invalidAppService' (matches the WebAppTreeItem better).